platform.md

```diff
@@ -645,10 +645,10 @@ Running `creator` SHALL be equivalent to running `detector`, `analyzer`, `restor
 | Input             | Environment Variable| Default Value| Description
 |-------------------|---------------------|--------------|----------------------
 | `<previous-image>`| `CNB_PREVIOUS_IMAGE`| `<image>`    | Image reference to be analyzed (usually the result of the previous build)
-| `<skip-restore>`  | `CNB_SKIP_RESTORE`  | `false`      | Do not write layer metadata or restore cached layers
+| `<skip-restore>`  | `CNB_SKIP_RESTORE`  | `false`      | Do not write layer metadata, restore sbom layers, or restore cached layers
```

Comment:
```suggestion
| `<skip-restore>`  | `CNB_SKIP_RESTORE`  | `false`      | Prevent buildpacks from reusing layers from previous builds, by skipping the restoration of any data to each buildpack's layers directory, with the exception of `store.toml`.
```

Maybe if we move away from describing the implementation it will be easier to understand and also give us a little more flexibility to make changes in our implementation.